source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -448,14 +448,31 @@ def similar_pt_section_line(section):
     return "pt:{}".format(section.pt_display_informations.uris.line)
 
 
-def similar_journeys_generator(journey, pt_functor):
+def similar_journeys_generator(journey, pt_functor, sn_functor=lambda mode: 'sn:{}'.format(mode)):
     for idx, s in enumerate(journey.sections):
         if s.type == response_pb2.PUBLIC_TRANSPORT:
             yield pt_functor(s)
         elif s.type == response_pb2.STREET_NETWORK and is_walk_after_parking(journey, idx):
             continue
         elif s.type in (response_pb2.STREET_NETWORK, response_pb2.CROW_FLY):
-            yield 'sn:%s' % s.street_network.mode
+            yield sn_functor(s.street_network.mode)
+
+
+def detailed_pt_section_vj(section):
+    return 'pt:{vj} dep:{dep} arr:{arr}'.format(
+        vj=section.pt_display_informations.uris.vehicle_journey,
+        dep=section.begin_date_time,
+        arr=section.end_date_time,
+    )
+
+
+def bss_walking_sn_functor(mode):
+    return None if mode in (FallbackModes.walking.value, FallbackModes.bss.value) else mode
```

Comment:
Would it not be better to return the `walking` mode instead ? 
Just wondering if a crow fly would not have it's mode set, it would be equal to a `walking` and a `bss`.